docs/user/installation_and_configuration.rst

```diff
@@ -0,0 +1,368 @@
+
+After GeoServer is running, you will also need to install the WPS plugin to your GeoServer
+instance. The GeoServer WPS Plugin (available at 
+http://docs.geoserver.org/stable/en/user/extensions/wps/install.html) must match the version of
+GeoServer instance. This is needed for both the Accumulo and Kafka variants of the plugin.
+
+For Accumulo
+^^^^^^^^^^^^
+
+To install the GeoMesa Accumulo GeoServer plugin, unpack the contents of the
+``geomesa-accumulo-gs-plugin-$VERSION.zip`` file in ``geomesa-$VERSION/dist/gs-plugins`` 
+into your GeoServer's ``lib`` directory (``$VERSION`` = |release|):
+
+If you are using Tomcat:
+
+.. code-block:: bash
+
+    $ unzip \
+      geomesa-$VERSION/dist/gs-plugins/geomesa-accumulo-gs-plugin-$VERSION-install.zip \
+      -d /path/to/tomcat/webapps/geoserver/WEB-INF/lib/
+
+If you are using GeoServer's built in Jetty web server:
+
+.. code-block:: bash
+
+    $ unzip \
+      geomesa-$VERSION/dist/gs-plugins/geomesa-accumulo-gs-plugin-$VERSION-install.zip \
+      -d /path/to/geoserver/webapps/geoserver/WEB-INF/lib/
+
+There are additional JARs for Accumulo, Zookeeper, Hadoop, and Thrift that will
+be specific to your installation that you will also need to copy to GeoServer's
+``WEB-INF/lib`` directory. For example, GeoMesa only requires Hadoop
+|hadoop_version|, but if you are using Hadoop 2.5.0 you should use the JARs
+that match the version of Hadoop you are running.
+
+There is a script in the ``geomesa-tools-$VERSION`` directory
+(``$GEOMESA_HOME/bin/install-hadoop-accumulo.sh``) which will install these
+dependencies to a target directory using ``wget`` (requires an internet
+connection). You may have to edit this file to set the versions of Accumulo,
+Zookeeper, Hadoop, and Thrift you are running.
+
+.. code-block:: bash
+
+    $ $GEOMESA_HOME/bin/install-hadoop-accumulo.sh /path/to/tomcat/webapps/geoserver/WEB-INF/lib/
+    Install accumulo and hadoop dependencies to /path/to/tomcat/webapps/geoserver/WEB-INF/lib/?
+    Confirm? [Y/n]y
+    fetching https://search.maven.org/remotecontent?filepath=org/apache/accumulo/accumulo-core/1.6.2/accumulo-core-1.6.2.jar
+    --2015-09-29 15:06:48--  https://search.maven.org/remotecontent?filepath=org/apache/accumulo/accumulo-core/1.6.2/accumulo-core-1.6.2.jar
+    Resolving search.maven.org (search.maven.org)... 207.223.241.72
+    Connecting to search.maven.org (search.maven.org)|207.223.241.72|:443... connected.
+    HTTP request sent, awaiting response... 200 OK
+    Length: 4646545 (4.4M) [application/java-archive]
+    Saving to: ‘/path/to/tomcat/webapps/geoserver/WEB-INF/lib/accumulo-core-1.6.2.jar’
+    ...
+
+If you do no have an internet connection you can download the JARs manually via http://search.maven.org/.
+These may include the JARs below; the specific JARs are included only for reference (assuming Accumulo 1.6.2,
+Zookeeper 3.4.5, Hadoop 2.2.0 and Thrift 0.9.1):
```

Comment:
Leaving this unchanged, as I followed the recommendation of the Accumulo GS plugin README.md (see @elahrvivaz 's comment above).